src/components/Navbar/Navbar.jsx

```diff
@@ -2,35 +2,61 @@ import React, { useContext } from "react";
 import "./Navbar.scss";
 import { Link } from "react-router-dom";
 import { StatusLengthContext } from "../../../StatusLengthContext";
-
+import { DarkModeContext } from "../../pages/ConfigurationPage/DarkModeContext";
 const Navbar = () => {
   const { todosCount } = useContext(StatusLengthContext);
+  const { darkMode } = useContext(DarkModeContext);
 
   return (
     <nav className="navbar">
       <ul className="filter-list">
         <li className="filter-item">
-          <Link className="navbar-item" to="/add">
+          <Link
+            className={
+              darkMode ? "navbar-item dark-mode" : "navbar-item light-mode"
```

Comment:
Write this condition once and use the class here. Something like:
```js
const navbarItemClass = darkMode ? "navbar-item dark-mode" : "navbar-item light-mode";
```
and then use `navbarItemClass`.

Reply:
+ I cannot see any class defined as `light-mode` in the css. What's the purpose of having that?

Reply:
Done.

Reply:
Done.